pkg/platform/kube/platform.go

```diff
@@ -145,6 +145,22 @@ func (p *Platform) CreateFunction(createFunctionOptions *platform.CreateFunction
 		return nil, errors.Wrap(err, "Create function options validation failed")
 	}
 
+	// it's possible to pass a function without specifying any meta in the request, in that case skip getting existing function
```

Comment:
I know you just moved this a bit, but this sounds dangerous...
It suggests a function exists and we should compare against it but because the user didn't fill meta.name we're gonna let that slide :\
Can you explain?

Reply:
I believe the original intention was to allow `PUT /api/functions/<name>` with partial body but since there both are required. I will add a TODO and resolve it on next PR